tentacle/src/channel/bound.rs

```diff
@@ -47,7 +48,10 @@ pub fn channel<T>(buffer: usize) -> (Sender<T>, Receiver<T>) {
 
     let tx = BoundedSenderInner {
         inner: inner.clone(),
+        #[cfg(not(feature = "parking_lot"))]
         sender_task: Arc::new(Mutex::new(SenderTask::new())),
+        #[cfg(feature = "parking_lot")]
+        sender_task: Arc::new(crate::lock::const_mutex(SenderTask::new())),
```

Comment:
I think there are no different than a normal `new` fn:
```suggestion
sender_task: Arc::new(Mutex::new(SenderTask::new()))
```